### DISCUSSION TITLE
Can't Open Quasar App on Browser

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#13164] [General - Components / Directives / etc]
Hi, I'm trying to open my project on localhost. When I try to open it in browser, it tries to download page. I can't see page. When I open downloaded file. It's index.template.html file. What the reason coulde be?

**Comment by mecjos** [comment]
This problem showed up after upgrade to quasar V2.

**Comment by metalsadman** [comment]
No error message? What vuerouter mode? History or hash?

**Reply by mecjos** [reply]
No error message, mode history. I was using quasar version 1.. I updated cli and created fresh project with vue3 and coppied old files into new project by making some changes on vue router and vuex. but it's weird, running local host via 'quasar dev' it works with only some warnings like '215:54  warning  Expected property shorthand  object-shorthand' this. try to open in browser it donwloads a file named 'download' without extension.

**Reply by mecjos** [reply]
by the way it was working, I deployed it and then upgraded quasar cli. then that's the situation.

**Reply by mecjos** [reply]
@metalsadman I added a gif to demonstrate problem.. it doesn't do it always and on every route you can see at gif.. do you have any idea about this issue?

https://ibb.co/0hLqVMs

**Reply by MilosPaunovic** [reply]
This indeed seem quite weird, but without a minimal code reproduction it would be hard for anyone to provide help better than just guessing, so creating one would be next logical step.

**Reply by mecjos** [reply]
@MilosPaunovic yes you are right.. that's very difficult to find by this way.. I noticed another thing, when I try to show page source on problematic route, it shows nothing.. but on page w/o problem it shows index.html page as expected... really weird.. but most probably this is related with quasar cli because this problem started to occur after cli update.

**Reply by metalsadman** [reply]
> @MilosPaunovic yes you are right.. that's very difficult to find by this way.. I noticed another thing, when I try to show page source on problematic route, it shows nothing.. but on page w/o problem it shows index.html page as expected... really weird.. but most probably this is related with quasar cli because this problem started to occur after cli update.

try to rollback to a version you think it was working fine and test again.

**Comment by mecjos** [comment]
Still couldn't fix the problem.. I thought browser problem but it's not... after sign-in it's same.. and it doesn't occur at nested routes..
